src/routes/firewall/cluster.js

```diff
@@ -191,9 +191,16 @@ router.post('/', async (req, res) => {
 	}
 
 	try {
+		let clusters = await Cluster.getClusterCloud(req)
+		if(clusters.length >= app().config.get('limits').clusters && app().config.get('limits').clusters>0) {
+			throw (fwcError.LIMIT_CLUSTERS)
+		}
```

Comment:
As it is a check when a new cluster is being created, it consults the clusters that are already created.
So if i remove `>=` and use `>` in case of having a limit of 1, it would let me put one more cluster.
That is, with a limit of 1, you could have 2 clusters.